README.md

```diff
@@ -173,7 +174,6 @@ If you like this project feel free to leave comments and contact via given autho
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-- [Microverse](https://www.microverse.org/)
 - [kimita]('https://github.com/kimitawanjohi')
```

Comment:
- [ ] Please make sure that you give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). :pray: